packages/story-editor/src/app/media/uploadQueue/index.ts

```diff
@@ -934,17 +991,17 @@ function useMediaUploadQueue() {
      * Checks whether the resource is not external and
      * not already uploading.
      *
-     * @param {import('@googleforcreators/media').Resource} resource Resource object.
-     * @return {boolean} Whether a given resource can be transcoded.
+     * @param resource Resource object.
+     * @return Whether a given resource can be transcoded.
      */
-    const canTranscodeResource = (resource) => {
-      const { isExternal, id, src } = resource || {};
+    const canTranscodeResource = (resource: Resource) => {
+      const { isExternal = false, id = '', src = '' } = resource || {};
```

Comment:
Why are there `|| {}` and default values here? The parameter is typed as required. If it's optional, it should be typed as `resource?: Resource` in the definition.

Reply:
Fixed